lib/phoenix/live_dashboard/helpers/tree_helpers/reingold_tifford_algorithm.ex

```diff
@@ -5,35 +5,36 @@ defmodule Phoenix.LiveDashboard.ReingoldTilford do
   @node_height 30
   @node_y_separation 10
   @total_y_distance @node_height + @node_y_separation
-
-  @node_width 120
   @node_x_separation 50
-  @total_x_distance @node_width + @node_x_separation
 
   def set_layout_settings(tree) do
     tree
     |> change_representation(0)
     |> calculate_initial_y(0, [])
     |> ensure_children_inside_screen()
     |> put_final_y_values(0)
+    |> put_x_position(:first_call)
   end
 
-  defp change_representation({{_, pid, _}, children}, level) do
+  defp change_representation({{_, pid, name}, children}, level) do
```

Comment:
Hi! So my suggestion is to keep this module completely independent from pid/name. So in here, we would do something like:

```suggestion
  defp change_representation({value, children}, level) do
```

And simply store it as `%{value: value}` in the ReingoldTifford tree. :) We can give meaning to the value later, when formatting. WDYT?

Reply:
Hi! sorry,  I was making other changes.
It means formatting the name of the node when `render(assigns)` is called?

Reply:
Yes! It should be part of the SVG generation.

Reply:
I need to format the name to calculate the node width and x position, I made some changes about it.

Reply:
Makes total sense. It is well encapsulated now anyway, so we easily clean it up later. :)